udata_gouvfr/theme/templates/header.html

```diff
@@ -4,6 +4,16 @@
         {{ _('Internet Explorer 8 and below are not anymore supported. Please upgrade or change your browser to access the website.') }} <a href="http://browsehappy.com/">{{ _('More information →')}}</a>
     </div>
     <![endif]-->
+
+    {% block banner %}
+      {% if config['BANNER_ACTIVATED'] %}
+        {% if config['BANNER_HTML_FILENAME'] %}
+          {% set b_filename = config['BANNER_HTML_FILENAME'] %}
+        {% endif %}
+        {% include theme('banners/{0}'.format(b_filename)) with context %}
```

Comment:
still not in the right `if` block (one above)

Reply:
I moved the `if` statement up a line, and simplified a bit this part given we have to specify a `BANNER_HTML_FILENAME` in udata... let me know if that's ok w/ U